src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+@import './variebles';
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    font-family: Roboto, sans-serif;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $secondary-text-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-family: Roboto, sans-serif;
+    font-weight: 400px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $blue-color;
+    border-radius: 5px;
+    color: $white-color;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    font-family: Roboto, sans-serif;
+  }
+
+  &__button:hover {
+    background-color: $white-color;
+    color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  .price-text {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: $secondary-text-color;
+    margin: 0;
+  }
+
+  .price-value {
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+    align-items: flex-end;
+    height: 16px;
+  }
+
+  .reviews-text {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  .container {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-image: url('../images/star.svg');
+}
```

Comment:
The `background-image` property is repeated in line 125. Consider using a modifier class or a pseudo-class to change the background image of the star when it's active.